packages/LUIS/bin/luis.js

```diff
@@ -49,6 +49,12 @@ async function runProgram() {
     if (argvFragment.length === 0) {
         argvFragment = ['-h'];
     }
+
+    let latest = await require('latest-version')(pkg.name);
+    if (semver.gt(latest, pkg.version)) {
+        process.stderr.write(chalk.default.yellowBright(`\nNew version ${latest} is available to install.\n\n`))
```

Comment:
Consider replacing this with these lines (reviewed text with design). 

        process.stderr.write(chalk.default.white(`\n     Update available`));
        process.stderr.write(chalk.default.gray(` ${pjson.version}`));
        process.stderr.write(chalk.default.white(` -> `));
        process.stderr.write(chalk.default.greenBright(`${latest}\n`));
        process.stderr.write(chalk.default.white(`       Run `));
        process.stderr.write(chalk.default.blueBright(`npm i -g ${pjson.name}`));
        process.stderr.write(chalk.default.white(` to update.\n\n`));